packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.h

```diff
@@ -97,6 +98,7 @@ typedef RCTBridge *_Nonnull (
 @interface RCTRootViewFactory : NSObject
 
 @property (nonatomic, strong, nullable) RCTBridge *bridge;
+@property (nonatomic, strong, nullable) RCTHost *reactHost;
```

Comment:
I'm good with this change, looks good - let's wait for @cipolleschi approval.

Reply:
It looks good to me too, but to reuse the `createReactHost` method, shouldn't it be exposed in the header as well, with:
```objc
- (RCTHost *)createReactHost;
```
?